cpp/tests/orderby/is_sorted/is_sorted_tests.cu

```diff
@@ -0,0 +1,239 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <gtest/gtest.h>
+#include <gmock/gmock.h>
+#include <cudf/cudf.h>
+#include <cudf/table.hpp>
+
+#include <utilities/error_utils.hpp>
+
+#include <tests/utilities/column_wrapper.cuh>
+#include <tests/utilities/column_wrapper_factory.hpp>
+#include <tests/utilities/cudf_test_fixtures.h>
+#include <tests/utilities/cudf_test_utils.cuh>
+
+template <typename T>
+using column_wrapper = cudf::test::column_wrapper<T>;
+
```

Comment:
Should probably have a test that verifies correct behavior for empty tables (0 columns or 0 rows).